tests/tango/test_tango_signals.py

```diff
@@ -27,6 +25,11 @@
 from tango.test_context import MultiDeviceTestContext
 from tango.test_utils import assert_close
 
+
+def __tango_signal_auto(*args, **kwargs):
+    raise RuntimeError("Fix this later")
```

Comment:
@burkeds I've removed tango_signal_auto as it was only used in `TangoDevice` creation of Signals that are not type hinted in the class. Is it used anywhere else? Can I delete the tests that go with them? Or replace them with `tango_signal_rw(None,...)` etc?

Reply:
tango_signal_auto is only used when automatically creating signals. Why would you want to delete it? Will you implement another method that does the same thing?